src/Model/LoginSession.php

```diff
@@ -229,15 +237,18 @@ public function getFriendlyUserAgent(): string
         $parser = Parser::create();
         $result = $parser->parse($this->UserAgent);
 
-        return sprintf('%s on %s', $result->ua->family, $result->os->toString());
+        return _t(
+            __CLASS__ . '.BROWSER_ON_OS',
+            "{browser} on {os}.",
```

Comment:
This just makes the string translatable. I used the `{}` syntax here because in some locale it might make more sense for the os to come first.